concepts/arrays/about.md

```diff
@@ -0,0 +1,143 @@
+# About
+
+JSON defines an array as:
+
+> An array is an ordered collection of values.
+> An array begins with `[` left bracket and ends with `]` right bracket.
+> Values are separated by `,` comma.
+
+Arrays can contain zero or more of _any kind_ of JSON value.
+
+## Creating arrays
+
+Use brackets to collect elements into an array
+```jq
+[1, 2, 3]
+```
+
+Inside the brackets can be an expression that generates a _stream_ of elements
```

Comment:
```suggestion
An expression inside brackets generates a _stream_ of elements
```

Reply:
Hmm, your edit is the opposite of what happens. How about

> The elements of a _stream_ can be captured into an array by enclosing it in brackets.
> ```jq
> range(5)     # => a stream of 5 numbers
> [range(5)]   # => the array [0, 1, 2, 3, 4]
> ```

Reply:
That makes much more sense to me!